### PR TITLE
Comments; ColorSensor; Nothing to do with vision tracking

### PR DESCRIPTION
**Pull request by p-casals**

### CODE REVIEW

**Comment by p-casals** [comment]
Got rid of all the pixypacket stuff; I'm not going to do that. ColorSensor works now and the comments are done

**Comment by p-casals** [comment]
intake buttons streamlined; they now make more sense. coloursensor can now take data automatically

File: src/main/java/frc/robot/commands/auto/routines/TestAutoCommandGroup.java [hunk 1]
```diff
@@ -6,10 +6,17 @@
...
+// TestAutoCommandGroup: For testing
 public class TestAutoCommandGroup extends SequentialCommandGroup {
+  
     public TestAutoCommandGroup(Drivetrain drivetrain) {
-        super(new MoveCommand(drivetrain, 24, 0.5),
-                new TurnCommand(drivetrain, 180, 0.5),
-                new MoveCommand(drivetrain, 24, 0.5));
+      
+        super(
+
```
Comment (LordSquishers) [inline src/main/java/frc/robot/commands/auto/routines/TestAutoCommandGroup.java]: heheh fix

File: src/main/java/frc/robot/subsystems/ColorSensor.java [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    colorMatchSupplier = () -> colorMatches;
+
+
+    /** 
+     * The below code is for spinning the control panel 4 times
+     */
+
+    String colorToStopOn = colorString;
+
+    for(int i = 0; i < 4; i++) {
```
Comment (LordSquishers) [inline src/main/java/frc/robot/subsystems/ColorSensor.java]: rethink

File: src/main/java/frc/robot/subsystems/Drivetrain.java [hunk 3]
```diff
@@ -46,6 +46,7 @@ public Drivetrain() {
     }
 
     public void tankDrive(double leftSpeed, double rightSpeed) {
+
```
Comment (LordSquishers) [inline src/main/java/frc/robot/subsystems/Drivetrain.java]: no

**Review by LordSquishers — CHANGES_REQUESTED** [review]

**Comment by p-casals** [comment]
alright done

**Review by LordSquishers — APPROVED** [review]
commenting out broken code does not fix it- please make sure this is functional

**Pull request merged**